cloud/jenkins/pgo-v2-operator-aws-openshift-latest.yml

```diff
@@ -0,0 +1,46 @@
+- job:
+    name: pgo-v2-operator-aws-openshift-latest
+    project-type: pipeline
+    description: |
+        Do not edit this job through the web!
+    triggers:
+      - timed: "0 6 * * 3"
```

Comment:
Please remove since we have a scheduler job which will schedule all "latest" jobs.